lib/kb_quast/kb_quastImpl.py

```diff
@@ -2,21 +2,23 @@
 #BEGIN_HEADER
 import errno as _errno
 import os as _os
+import subprocess as _subprocess
 import time as _time
+import uuid
 import uuid as _uuid
-import subprocess as _subprocess
-from Workspace.WorkspaceClient import Workspace as _WSClient
-from Workspace.baseclient import ServerError as _WSError
-from DataFileUtil.DataFileUtilClient import DataFileUtil as _DFUClient
-from DataFileUtil.baseclient import ServerError as _DFUError
-from AssemblyUtil.AssemblyUtilClient import AssemblyUtil as _AssClient
-from AssemblyUtil.baseclient import ServerError as _AssError
-from KBaseReport.KBaseReportClient import KBaseReport as _KBRepClient
-from KBaseReport.baseclient import ServerError as _RepError
+
 import psutil
-import uuid
 from Bio import SeqIO as _SeqIO
 
+from installed_clients.AssemblyUtilClient import AssemblyUtil as _AssClient
+from installed_clients.DataFileUtilClient import DataFileUtil as _DFUClient
+from installed_clients.KBaseReportClient import KBaseReport as _KBRepClient
+from installed_clients.WorkspaceClient import Workspace as _WSClient
+from installed_clients.baseclient import ServerError as _AssError
```

Comment:
There's no real reason to import the same thing into different names. The previous code had to deal with the fact that each of the ServerErrors were different classes with the same unqualified name, and so couldn't import them all as Server error or only the last import would be valid, and most of the `excepts` would fail to work because it was trying to catch the wrong class.